testing/python/approx.py

```diff
@@ -496,3 +496,13 @@ def test_numpy_scalar_with_array(self):
         assert actual != approx(expected, rel=5e-8, abs=0)
         assert approx(expected, rel=5e-7, abs=0) == actual
         assert approx(expected, rel=5e-8, abs=0) != actual
+
+    def test_generic_iterable_sized_object(self):
+        class newIterable(object):
```

Comment:
I suggest using a `My` prefix (`MyIterable`) which is used in a few other places in our tests, and also follows the CamelCase naming convention for classes. 👍